vm/devices/firmware/firmware_uefi/src/service/diagnostics.rs

```diff
@@ -462,12 +468,14 @@ impl UefiDevice {
     /// # Arguments
     /// * `allow_reprocess` - If true, allows processing even if already processed for guest
     /// * `context` - String to indicate who triggered the diagnostics processing
-    pub(crate) fn process_diagnostics(&mut self, allow_reprocess: bool, context: &str) {
-        if let Err(error) = self.service.diagnostics.process_diagnostics(
-            allow_reprocess,
-            &self.gm,
-            handle_efi_diagnostics_log,
-        ) {
+    pub(crate) fn process_diagnostics(&mut self, allow_reprocess: bool, limit: u32, context: &str) {
```

Comment:
do we want to use context when processing is successful? might we want to more easily know what happened?